src/permissions.rs

```diff
@@ -56,13 +59,15 @@ macro_rules! permissions {
             )*
             #[serde(default)]
             bors: BorsPermissions,
+            crates_io_ops_apps: CratesIoOps,
```

Comment:
Nit: I'd call the permission `crates_io_ops_bot.APP` instead of `crates_io_ops_apps.APP`.